pkg/serverconfig/config.go

```diff
@@ -35,10 +35,21 @@ func SetFlagsFromConfig(fs *flag.FlagSet, filename string) (err error) {
 	addAuth(fs, &config.Auth)
 	addCustomization(fs, &config.Customization)
 	addProviders(fs, &config.Providers)
+	addHelmConfig(fs, &config.Helm)
 
 	return nil
 }
 
+func addHelmConfig(fs *flag.FlagSet, helmConfig *Helm) (err error) {
```

Comment:
Note that this will have to be in sync with the operator config builder if its going to become public API:
https://github.com/openshift/console-operator/blob/master/pkg/console/subresource/consoleserver/config_builder.go#L1